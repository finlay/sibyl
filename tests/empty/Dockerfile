# Generated by sibyl

FROM stencila/alpha
