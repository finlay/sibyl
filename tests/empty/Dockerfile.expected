# Generated by sibyl

FROM stencila/alpha
COPY stencila.tar.gz stencila.tar.gz
RUN tar xf stencila.tar.gz && rm stencila.tar.gz
